backend/testUtils/testDb.ts

```diff
@@ -8,7 +8,7 @@ const mongoTest = {
   connect: async (): Promise<void> => {
     await mongo.start();
     const uri = mongo.getUri();
-    await mongoose.connect(uri);
+    await mongoose.connect(uri, { useNewUrlParser: true });
```

Comment:
I don't think this is needed anymore: https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options